lib/github/github.ex

```diff
@@ -183,7 +183,7 @@ defmodule BorsNG.GitHub do
   def get_user_by_login!(token, login) do
     {:ok, user} = GenServer.call(
       BorsNG.GitHub,
-      {:get_user_by_login, token, {login}},
+      {:get_user_by_login, token, {String.trim(login)}},
```

Comment:
I only trimmed in this place because I am sure GH does not allow such so I will consider it a typo. 

On the other hand, I'm not doing any assumption on any other user input

Reply:
Just to make sure, when you do this, you've made sure that it can't result in us accidentally creating a record in the `users` table with a leading space?

Reply:
Good catch, will confirm it and add a maybe-redundant trimmer to the users model

Reply:
On all cases that we use this function, we use it's return as the username when inserting the record. This makes this problem something that _should_ not happen (unless we change the code somehow without noticing it) and also by incident handles the potential bug with casing (`"NOTriddle" =/= "notriddle"`)

Reply:
So we can either:
a) leave it with the trim to handle the case where the user copy-paste with an space
b) remove this trim and put an additional line on the error saying that it might have occurred by the present of whitespace

Other than that, LGTM :)

Reply:
Nah, let's just go ahead and merge as-is.